src/main/java/org/jabref/gui/preferences/XmpPrivacyItemModel.java

```diff
@@ -0,0 +1,29 @@
+package org.jabref.gui.preferences;
+
+import javafx.beans.property.ObjectProperty;
+import javafx.beans.property.SimpleObjectProperty;
+
+import org.jabref.model.entry.field.Field;
+
+public class XmpPrivacyItemModel {
+    private ObjectProperty<Field> field;
+
+    XmpPrivacyItemModel(Field field) {
+        this.field = new SimpleObjectProperty<>(field);
+    }
+
+    public void setField(Field field) { this.field.setValue(field); }
+
+    public Field getField() { return this.field.getValue(); }
+
+    public ObjectProperty<Field> fieldProperty() {
+        return field;
+    }
+
+    public String getName() { return field.getValue().getName();}
```

Comment:
What about using `getDisplayName` instead of `getName`?

Reply:
Changed it, but in the new FieldsUtil::getNameWithType.